app.py

```diff
@@ -390,239 +188,31 @@ def snap_details(snap_name):
 
 # Publisher views
 # ===
-@app.route('/account/snaps/<snap_name>/measure')
+@app.route('/account')
```

Comment:
Would it be possible in flask to 'delegate' part of the routing to separate module?

I mean - with this PR even that all the publisher views are in separate module it's `app.py` that defines all URLs for them.
Would it be possible to say: everything in `/account` is handled by publisher module, and then the publisher module will define specific publisher URLs?

Reply:
As discussed, for now we are going to keep the route in app to keep the app simple. If we feel that we'll need to move the routes on their own modules we will create a new Refactoto Season 😄